src/components/toast/toast.ts

```diff
@@ -86,6 +92,13 @@ export class Toast extends ViewController {
   }
 
   /**
+  * @private
+  */
+  isVaidPosition(position: string) {
```

Comment:
I fixed it after merging :smile: https://github.com/driftyco/ionic/commit/c67ce8c906cfaebe0e591bd43094342690fdb43c